cypress/bin/run-e2e

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+rm -rf cypress/webspace/vendor/contao/manager-bundle
+composer up --working-dir=cypress/webspace
+
+symfony server:start --port=8765 --dir=cypress/webspace --daemon
+yarn cypress run
+symfony server:stop --dir=cypress/webspace
```

Comment:
Are the paths correct here? 
Wouldn’t it have to be something like `../webspace/vendor/contao/manager-bundle` instead?

Reply:
Paths are correct as the script is called from the base directory (`yarn run e2e-tests`).